src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -135,7 +149,7 @@ const LaunchAnalysisButton = () => {
   const renderLaunchButton = () => {
     let buttonText;
 
-    if (pipelineRerunStatus.rerun) {
+    if (pipelinesRerunStatus.gem2s.rerun || pipelinesRerunStatus.qc.rerun) {
```

Comment:
You can just have a single variable `pipelineToRerun` which is determined to be `gem2s`, `qc` or `null` and can be used to simplify the logic in other places. 
Like on like 178  - if (pipelineToRerun) instead of if (pipelinesRerunStatus.gem2s.rerun || pipelinesRerunStatus.qc.rerun)

And next line const rerunPipelineType = pipelinesRerunStatus.gem2s.rerun ? 'gem2s' : 'qc'; can be just const rerunPipelineType = pipelineToRerun

Reply:
good point